src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -305,25 +305,25 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
           isLoading={isEventsLoading}
           {...scheduleNGWithAttendees}
         />
-      </div>
-      {!isEventsLoading && (
-        <div className="NavBarWrapper__download-buttons">
-          {isShowPersonalDownloadBtn && (
+        {!isEventsLoading && (
+          <div className="NavBarSchedule__downloadButtons">
+            {isShowPersonalDownloadBtn && (
+              <Button
+                onClick={downloadPersonalEventsCalendar}
+                customClass="NavBarSchedule__downloadScheduleBtn"
+              >
+                Download your schedule
+              </Button>
```

Comment:
Could we use `<ButtonNG/>` component here?

Reply:
Any instance where `Button` can be replaced with `ButtonNG` that doesn't require significant effort for adjustment is a plus. The sooner we eliminate the need of  `Button`, the sooner we can get rid of some `global.scss` styles that are hard to override.

Reply:
fixed in 455ec1701e65eecc756037aea9c0f4502148a8e7